api-get-options.go

```diff
@@ -53,6 +54,11 @@ func (o GetObjectOptions) Header() http.Header {
 	if o.ServerSideEncryption != nil && o.ServerSideEncryption.Type() == encrypt.SSEC {
 		o.ServerSideEncryption.Marshal(headers)
 	}
+	// this header is set for active-active replication scenario where GET/HEAD
+	// to site A is proxy'd to site B if object/version missing on site A.
+	if o.Internal.ReplicationProxyRequest {
```

Comment:
Will this be needed by MinIO server replication code ?

Reply:
yes